Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -190,6 +194,8 @@ open class MapView: UIView {
 
         dependencyProvider = MapViewDependencyProvider(interfaceOrientationProvider: orientationProvider)
         attributionUrlOpener = DefaultAttributionURLOpener()
+        applicationStateProvider = DefaultApplicationStateProvider()
```

Comment:
I wonder if it would make more sense to use `UIApplicationApplicationStateProvider` and `UIApplicationPreferredContentSizeCategoryProvider` instead since this is marked as iOSApplicationExtension unavailable.

Reply:
I used `Default` to align with the original PR. We should get guidance from the Mapbox team on this one.